charts/ocis/values.yaml

```diff
@@ -530,7 +545,9 @@ secretRefs:
   machineAuthApiKeySecretRef: ""
   # -- Reference to an existing messaging system certificate authority secret (see ref:Secrets#secrets)
   messagingSystemCaRef: ""
-  # -- Reference to an existing SMTP email server settings secret (see ref:Secrets#secrets). Not used if `features.emailNotifications.enabled` equals `false`.
+  # -- Reference to an existing SMTP email server settings secret (see ref:Secrets#secrets).
+  # Not used if `features.emailNotifications.enabled` equals `false`.
+  # Not used if `features.emailNotifications.smtp.authentication` equals `none`.
```

Comment:
I think this description needs some more information about the keys we expect in the secret. Not especially related to this PR this was already missing before.

Reply:
Oh, the secret section https://doc.owncloud.com/ocis/next/deployment/container/orchestration/orchestration.html#secrets needs a complete rework since we switched to automatically creating secrets. Tracking issue: #367